testing/fs-uae/APKBUILD

```diff
@@ -14,8 +14,6 @@ source="https://fs-uae.net/stable/${pkgver}/${pkgname}-${pkgver}.tar.gz"
 options="!check"
 
 build() {
-	export CC=clang
-	export CXX=clang++
```

Comment:
I guess we won't need `clang-dev` as makedepend then?

Reply:
Indeed. Fixed.

Reply:
Thanks